.github/workflows/pr.yml

```diff
@@ -0,0 +1,54 @@
+---
+    name: Build and test PR
+
+    permissions:
+      contents: read
+      packages: write
+      # To report GitHub Actions status checks
+      statuses: write
+      security-events: write
+      id-token: write
+
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
+      cancel-in-progress: true
+
+    on:
+      pull_request:
+        types:
+          - opened
+          - synchronize
+          - ready_for_review
+          - reopened
+        branches:
+          - main
+
+    jobs:
+      lint:
+        name: Lint
+        runs-on: ubuntu-latest
+        permissions:
+          contents: read
+          packages: read
+          # To report GitHub Actions status checks
+          statuses: write
+
+        steps:
+          - uses: actions/checkout@v4
+            with:
+              # super-linter needs the full git history to get the
+              # list of files that changed across commits
+              fetch-depth: 0
+              submodules: true
+
+          - name: Run ansible-lint
+            uses: ansible/ansible-lint@v25.4.0
+
+          - name: Run super-linter
+            uses: super-linter/super-linter@v7.3.0
+            env:
+              GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+              VALIDATE_JSCPD: false
```

Comment:
Please could you move these to `super-linter.env`, and populate the GitHub environment like in this commit: https://github.com/azimuth-cloud/azimuth-images/pull/316/commits/2d3234bbf81b04cdc47e9db3bf4409e3c52a14e4